docs/howtos/customization/create_custom_role.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: "Creating A Git Configuration"
```

Comment:
To confirm, was it meant this or instead "Creating a custom role"?

Reply:
Ah, yes. Copy-pasta :spaghetti: